contracts/rewards/base/emission_manager.cairo

```diff
@@ -81,12 +89,13 @@ end
 
 @external
 func set_emission_admin{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
-    reward : felt, admin: felt
+    reward : felt, admin : felt
 ):
     Ownable.assert_only_owner()
-    let (old_admin) = _emission_admins.read(reward)
+    let (_old_admin) = _emission_admins.read(reward)
```

Comment:
no need to use prefix: prefix `_` is a Solidity convention for storage variables. currently, the convention we use if: storage variable are prefixed with the name of the namespace. see https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/Extensibility.md#libraries